### DISCUSSION TITLE
Looking for a way to reset the current pane

### DISCUSSION THREAD

**Discussion by themadsens** [wezterm/wezterm#2606] [Q&A] [answered]
Could not find anything in the docs for binding keys to a "Reset pane" function
I tried sending [ESC-c](https://wezfurlong.org/wezterm/escape-sequences.html?highlight=ris#other-escape-sequences) on the pane, but neither :paste() nor :send_text really serves the purpose.
I could use `pane:send_text("printf '\\ec'\n")` but that would have a rather limited utility. (Only work at the bare prompt)

**Comment by wez** [comment, marked as answer]
4e5945c06190478adbd987fa64a0bdbebf72054b adds a `ResetTerminal` key assignment action that can be used for this.
It also adds `pane:inject_output(string)` which can be used to send arbitrary output to the output side of the terminal emulator and have it interpret any escapes contained in that text

**Reply by wez** [reply]
It typically takes about an hour before commits are available as nightly builds for all platforms.  Linux builds are the fastest to build and are often available within about 20 minutes.  Windows and macOS builds take a bit longer.

Please take a few moments to try out the change and let me know how that works out. You can find the nightly downloads for your system in the [wezterm installation docs](https://wezfurlong.org/wezterm/installation.html).

If you prefer to use packages provided by your distribution or package manager of choice and don't want to replace that with a nightly download, keep in mind that you can download portable packages (eg: a `.dmg` file on macOS, a `.zip` file on Windows and an `.AppImage` file on Linux) that can be run without permanently installing or replacing an existing package, and can then simply be deleted once you no longer need them. 

If you are eager and can [build from source](https://wezfurlong.org/wezterm/install/source.html) then you may be able to try this out more quickly.

**Reply by themadsens** [reply]
Wow that was fast. Actually I just saw it in a git pull. Will try asap :)

**Reply by themadsens** [reply]
Both work like a charm :) Thanks
